app/views/lks/content.html

```diff
@@ -86,7 +86,12 @@ <h2>Copyright Act & 'Library Privilege'</h2>
         </section>
       </div>
 
-      <div class="nhsuk-grid-column-one-third">
+      <div class="nhsuk-grid-column-one-third">     
+        
+        <div class="nhsuk-sidebar-image">
+          <img src="/images/placeholders/dental-region-map.png" alt="Dental region map" />
+        </div>
```

Comment:
Probably overkill for this, but would it be possible to have this rendered using nunjucks to keep consistent with other front end components? Not sure how familiar you are with nunjucks, so may be a nice simple one to play around with.